scripts/utilities/example_pyrescal

```diff
@@ -0,0 +1,71 @@
+#!/usr/bin/env python3
```

Comment:
Add `.py` extension to filename

Reply:
This example is meant to be used by a job manager. 
For example:
 srun --wait=0 --cpus-per-task=2 --ntasks=8 example_pyrescal

Reply:
Comments added to example_pyrescal to clarify. Also documentation added for this example in docs.

Reply:
The extension doesn't prevent the job manager from using the script, but makes it easy for humans to identify its nature.